nntrainer/tensor/float_tensor.h

```diff
@@ -178,6 +178,30 @@ class FloatTensor : public TensorBase {
   TensorV2 &apply(std::function<float(float)> f,
                   TensorV2 &output) const override;
 
+  /**
+   * @copydoc TensorV2::multiply_i(float const &value)
+   * @todo    Need implementation and unit tests
+   */
+  int multiply_i(float const &value) override { return 0; }
+
+  /**
+   * @copydoc TensorV2::multiply(float const &value, TensorV2 &out)
+   * @todo    Need implementation and unit tests
+   */
+  TensorV2 &multiply(float const &value, TensorV2 &out) const override {
+    return out;
```

Comment:
What if we add "throw std::logic_error("Not yet implemented");" in case implementation is not done yet? Not in this PR, but perhaps in a future PR.

Reply:
good idea! I'll add it in later PRs with skeleton codes :)